docs/users.rst

```diff
@@ -26,20 +41,157 @@ image owners can use ``koji watch-logs <task id> | grep -w x86_64``.
 Configuring koji-containerbuild for koji CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+To be able to build container images from CLI `koji-containerbuild plugin`_
+must be installed.
+
+To configure CLI you'll need to copy [koji] section in /etc/koji.conf to
+[koji-containerbuild] and optionally adapt configuration there.
+
+.. _`koji-containerbuild plugin`: https://github.com/containerbuildsystem/koji-containerbuild
+
+
+Koji Build Results
+~~~~~~~~~~~~~~~~~~
+
+Koji Web
+********
+
+This is the easiest way to access information about OSBS builds.
+
+List Builds
++++++++++++
+
+Navigate to the “Builds” tab in brew and set the “Type” filter to ``image``.
+
+
+Get Build
++++++++++
+
+If you have the build ID, got to https://<KOJI_WEB_URL>/buildinfo?buildID=<build-ID>
+
+If you want to search by prefix, use the search box on top of the page.
+For example, redis-* and select “Builds”.
+
+This will also show the tarball docker image produced by build.
```

Comment:
In koji build you can find a lot information about build, some noticeable are : 
- pull specifications for the build in the 'Extra' section image.index.pull, for digest type in image.index.digests
- list of image archives for each specific architecture for which build was executed (for more detailed information about specific archive click on 'info'


# I am not so sure if it is useful to talk about docker.repositories in archive, because that is just for v2s2 image, and usually people are more interested in pull spec for manifest_list since with that they can pull image to any architecure